charts/deconz/templates/hpa.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```

Comment:
Given that this works by connecting to a physical device, does an HPA make sense here?  What's the behavior or scenario where more than one replica would be employed?

Reply:
Fair question... My rationale for leaving the HPA in there (it was generated by `helm create`) is to be able to enforce a hard upper limit on number of replicas. When doing a `kubectl scale` with a number of replicas that exceeds  `maxReplicas` it will automatically be scaled down to that number (in this case 1)

Reply:
Clever approach!